### DISCUSSION TITLE
Feature suggestion: Open [...] in instance

### DISCUSSION THREAD

**Discussion by Zorakie** [arnoudkooi/SN-Utils#327] [Ideas]
So this idea comes to me while working on deployments:

Often, I need to compare the records between instances. Doing it manually - copy url, open tab, paste, navigate to the beginning rewrite.

Wouldn't be nice, if you could do that in 2 clicks?

How to do it:
1) In settings, new section will be created, can be called "Environments". Each line represents the "environment", each column represents the instace
In another words, every line can represent e.g. the customer you are working for, each column his instances, dev/test/preprod/prod.

2) Into the context menu of the webpage under Tools, add the item, that will open another submenu with the instances that are within the same environment, but not the instance you are on right now (In another words, if you are on mycustomerdev, right_click->SNUtils->Tools-> Show in instance will be the environment options - mycustomertest and mycustomerprod (showing the instance domain as label.
Any other urls from other "environments" will not be shown.

For convenience, there can be a new tab in the SNUtils window, that will simply dynamically print available instance buttons for that environment, same as in the context menu.

Under the hood, it won't be nothing more than domain rewrite.
Needs to work for most of the links, mainly forms/list, if you want to compare specific form or just the list of records, if the numbers match.

So what do you think?

**Comment by arnoudkooi** [comment]
Are you aware of the commands 
`/env instancename` Opens current record in other instance [tweet](https://twitter.com/sn_utils/status/1174825829018324992)
and
`/diffenv instancename` compares XML of both current records [tweet](https://twitter.com/sn_utils/status/1527640837420683266)

The new arrow up may help as well, also check the cheatsheet and video
https://www.arnoudkooi.com/cheatsheet/

**Comment by arnoudkooi** [comment]
Hope the suggestion worked for you...

**Comment by Zorakie** [comment]
Hi, sorry, yes, I was not aware of that, It covers the needs. Thank you for your support!
